package.json

```diff
@@ -73,7 +73,7 @@
     "http-proxy-agent": "2.1.0",
     "https-proxy-agent": "2.2.1",
     "jsonc-parser": "1.0.0",
-    "microsoft.aspnetcore.razor.vscode": "https://download.visualstudio.microsoft.com/download/pr/43cb2b1b-4703-48d6-9e08-c57ffc92ce5e/53d28ad0bbbee4d7d0ac7cb2bf575708/microsoft.aspnetcore.razor.vscode-1.0.0-alpha1-20180925.1.tgz",
+    "microsoft.aspnetcore.razor.vscode": "https://razorvscodetest.blob.core.windows.net/npm/microsoft.aspnetcore.razor.vscode-1.0.0-alpha1-20180928.2.tgz",
```

Comment:
Can undo this, wasn't sure how to get bits uploaded to the CDN so just updated to use the dev bits.